exe/Main.hs

```diff
@@ -1,5 +1,6 @@
 -- Copyright (c) 2019 The DAML Authors. All rights reserved.
 -- SPDX-License-Identifier: Apache-2.0
+{-# LANGUAGE CPP               #-}
```

Comment:
What we've been doing elsewhere is just to use the old imports and accept the deprecation warnings. Up to you, but I think the CPP is more painful than the warning.

Reply:
The `CPP` is more forward compatible and sets an example on how to do it